emcache/client.py

```diff
@@ -352,7 +361,11 @@ async def gets(self, key: bytes, return_flags=False) -> Optional[Item]:
             else:
                 return await self._autobatching_flags_cas.execute(key)
 
-        keys, values, flags, cas = await self._fetch_command(b"gets", key)
+        result = await self._fetch_command(b"gets", key)
+        if len(result) != 4:
```

Comment:
I guess that this what I do not like, to have to check the len of the result.

The other option would be, to just return 5 parameters, like

```
keys, values, flags, cas, client_error = result

if client_error:
    raise CommandError(client_error)
```

WDYT?

Reply:
Golang)

Reply:
so?